appdatabase/migrations/sql/1721215212_create_keycard_and_accounts.up.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO keycards (keycard_uid, keycard_name, keycard_locked, key_uid, position)
+SELECT
+    s.keycard_instance_uid,
+    s.name,
+    0 AS keycard_locked,
+    s.key_uid,
+    0 AS position
+FROM settings s
+WHERE s.keycard_instance_uid IS NOT NULL
+  AND s.keycard_instance_uid NOT IN (SELECT keycard_uid FROM keycards);
+
+INSERT INTO keycards_accounts (keycard_uid, account_address)
```

Comment:
For profile key pairs that have already been migrated to a keycard, this query as it's now, will add one more row for the default wallet account, even if it has already been added.

Reply:
thank you @saledjenic good point, could please suggest how it could be done properly?

Reply:
actually this will work only for v1 profiles which wasn't migrated, because of this condition `AND kpa.key_uid IN (SELECT key_uid FROM settings WHERE keycard_instance_uid IS NOT NULL)`

Reply:
Unfortunately `keycard_instance_uid` is set for all users who migrated their profile to a keycard.

Reply:
> thank you @saledjenic good point, could please suggest how it could be done properly?

@flexsurfer try this:
```
INSERT INTO keycards_accounts (keycard_uid, account_address)
SELECT
    k.keycard_uid,
    kpa.address
FROM keypairs_accounts kpa
JOIN keycards k ON k.key_uid = kpa.key_uid
WHERE kpa.chat = 0
  AND kpa.key_uid IN (SELECT key_uid FROM settings WHERE keycard_instance_uid IS NOT NULL)
  AND NOT EXISTS (
      SELECT 1
      FROM keycards_accounts ka
      WHERE ka.account_address = kpa.address
  );
```